llama_index/llms/openai_like.py

```diff
@@ -0,0 +1,31 @@
+from typing import Any
+
+from llama_index.bridge.pydantic import Field
+from llama_index.llms import OpenAI
+
+
+class OpenAILike(OpenAI):
```

Comment:
Can you add a class docstring just basically restating the ideas you have from the PR description?